zhaquirks/xiaomi/aqara/switch_acn047.py

```diff
@@ -74,6 +74,14 @@ def _update_attribute(self, attrid, value):
             super()._update_attribute(attrid, value)
 
 
+class T2MeteringCluster(MeteringCluster):
+    """T2 Metering cluster to fix divisor."""
+
+    _CONSTANT_ATTRIBUTES = {
+        MeteringCluster.AttributeDefs.divisor.id: 1000000,
+    }
```

Comment:
To not remove all other constants attribute, they'll need to be added again:
```suggestion
    _CONSTANT_ATTRIBUTES = {
        **MeteringCluster._CONSTANT_ATTRIBUTES,
        MeteringCluster.AttributeDefs.divisor.id: 1000000,
    }
```
(You might not notice anything breaking when not doing this, but that's because the other attributes would have been read during pairing, where they are cached.)